src/js/Controllers/RpcFactory.js

```diff
@@ -23,6 +23,19 @@ class RpcFactory {
         }
         return msg;
     }
+    static ErrorResponse(rpc, code, info) {
```

Comment:
Might be a better idea to modify the existing `UIPerformInteractionFailure` method.

Reply:
Eh, I find it a bit more confusing that there isn't a more generic error response method. If you want, I could modify that method to use this one, but I don't think this should be removed